packages/Sortable/src/components/Item/Item.styles.js

```diff
@@ -0,0 +1,65 @@
+import tokens from "@paprika/tokens";
+import stylers from "@paprika/stylers";
+
+const draggingStyles = `
+  background: ${tokens.color.white};
+  box-shadow: ${tokens.shadow}, 0 0 1px ${tokens.border.color};
+
+  &:focus {
+    box-shadow: ${tokens.shadow}, ${tokens.highlight.active.withBorder.boxShadow};
+  }
+`;
+
+export const itemStyles = props => `
```

Comment:
wouldn't be more "styled-components" approach to use **css``** instead of pass a function with props? the previous can be confused with a definition of a Functional component.

Reply:
Hm, I've been using this technique in a few components now, and like I mentioned in a previous comment, like how it cleans up the conditions inline with the reset of the css.  I doubt it will be confused with a functional component, given the context of it being found in a `styles.js` file, and it's body being a template literal.

Reply:
Do we get syntax highlighting here?

Reply:
Nopes. I don't get proper syntax highlighting ever with `styled-components`.

Reply:
ref: https://github.com/acl-services/paprika/pull/92#discussion_r296359081